File: java/questao2.java
import java.util.Scanner;

public class questao2 {

    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);
        System.out.print("Digite o um numero: ");
        int n1 = scan.nextInt();
        if (n1%2 == 0){
            System.out.print("O número digitado é par.");
        }else{
            System.out.print("O número digitado é impar.");
        }
}
}

File: questao2.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int numero1;
    printf("Digite um numero: ");
    scanf("%d", &numero1);

    if(numero1%2==0){
      printf("O numero digitado e par");

    } else{
        printf("O numero digitado e impar");

    }
}